doc/book/helpers/body-parse.md

```diff
@@ -43,13 +43,17 @@ return [
         ],
     ],
     'middleware_pipeline' => [
-        'pre_routing' => [
-            [ 'middleware' => Helper\BodyParams\BodyParamsMiddleware::class ],
-            /* ... */
-        ],
-        'post_routing' => [
-            /* ... */
+        [ 'middleware' => Helper\BodyParams\BodyParamsMiddleware::class, 'priority' => 100],
```

Comment:
There are a few places it could be pushed:

- prior to routing (demonstrated)
- between routing and dispatch (not demonstrated)
- in routing-specific middleware pipelines (demonstrated, and listed as the optimal path, as it ensures it's not executed unless your middleware is actually expecting body params)

We can add the middle one via another PR later; I merely updated the existing examples to the changes in this PR.